scripts/release_scripts/update_changelog_and_credits_test.py

```diff
@@ -1,4 +1,3 @@
-# coding: utf-8
 #
```

Comment:
Should we also remove this line?

Reply:
It was unintentional, added the line back.